modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractTypeScriptClientCodegen.java

```diff
@@ -274,39 +283,36 @@ public String modelFileFolder() {
 
     @Override
     public String toParamName(String name) {
-        // sanitize name
         name = sanitizeName(name, "[^\\w$]");
 
         if ("_".equals(name)) {
             name = "_u";
         }
 
-        // if it's all uppper case, do nothing
-        if (name.matches("^[A-Z_]*$")) {
```

Comment:
I carelessly removed this special case.
Please let me know if there is a particular reason to handle upper case differently from everything else!